CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] PromQL: ignore small errors for bucketQuantile #6766
+* [BUGFIX] Fix issue where queries can fail or omit OOO samples if OOO head compaction occurs between creating a querier and reading chunks #6766
+* [BUGFIX] Fix issue where concatenatingChunkIterator can obscure errors #6766
+* [BUGFIX] Fix panic during tsdb Commit #6766
+* [BUGFIX] tsdb/head: wlog exemplars after samples #6766
```

Comment:
These are all very technical descriptions of these changes which may not make sense to a user of Mimir. Perhaps something like this would be clearer? (don't trust my descriptions of these - please check them with the people that made these changes)

```suggestion
* [ENHANCEMENT] Querier: ignore floating point precision issues when computing histogram quantiles. #6766
* [BUGFIX] Fix issue where queries can fail or omit OOO samples if OOO head compaction occurs while the query is running. #6766
* [BUGFIX] Fix panic while writing samples. #6766
* [BUGFIX] ??? #6766
```